src/python/espressomd/MDA_ESP/__init__.py

```diff
@@ -164,6 +178,7 @@ def parse(self):
                  Tempfactors(np.zeros(natoms)),
                  ICodes(np.array([' '], dtype=object)),
                  Charges(np.array(charges)),
+                 Bonds(bonds)
```

Comment:
Should the angles and dihedrals be here too?

Reply:
yes, it's already fixed in the branch, but GitHub is not updating it (see 03a07713ae)